lib/beaker/host_prebuilt_steps.rb

```diff
@@ -561,16 +561,9 @@ def set_env host, opts
 
       block_on host do |host|
         skip_msg = host.skip_set_env?
-        if skip_msg.nil?
-          env = construct_env(host, opts)
-          logger.debug("setting local environment on #{host.name}")
-          if host['platform'] =~ /windows/ and host.is_cygwin?
-            env['CYGWIN'] = 'nodosfilewarning'
-          end
-          host.ssh_permit_user_environment
-          host.ssh_set_user_environment(env)
-        else
-          logger.debug(skip_msg)
+        unless skip_msg.nil?
+            logger.debug( skip_msg )
+            next
```

Comment:
@trevor-vaughan it looks like the spec failures are caused by the fact that although it was a great idea to put the guard clause here first & get the error case out of the way, the main code path has been erased when I assume it should be just below the guard clause.

Reply:
@kevpl Ah, I see. I won't have time to get to this before Sunday so feel free to just update the PR directly (I have allow maintainers to update) if you want to move it forward.